docs/AzureADB2C.md

```diff
@@ -0,0 +1,185 @@
+# Overview
```

Comment:
Are you referring to how we couldn't get the app service authentication working an instead opted for AD B2C?  If yes - I would rather not since Azure AD B2C is a better overall solution for a production application.